.github/PULL_REQUEST_TEMPLATE

```diff
@@ -33,7 +32,7 @@ Add mentions of things that are not covered here and are planed to be done in se
 Merge pull request #123 from twisted/4356-branch-name-with-trac-id
 
 Author: <github_username>, <github_usernames_if_more_authors>
-Reviewer: <github_username>, <github_usernames_if_more_reviewers>
+Reviewer: <github_username>, @<github_username_if_more_reviewers>
```

Comment:
`@s` in commit messages can be *very annoying*

People can fork projects and rebase a lot of commits - firing off thousands of notifications

Reply:
Thanks for the feedback. I was not aware of this. 
Autocompletion is nice... but I can remove the `@` at the end :)